src/utils/hsdis/capstone/hsdis-capstone.c

```diff
@@ -150,6 +150,9 @@ void* decode_instructions_virtual(uintptr_t start_va, uintptr_t end_va,
   Options ops = parse_options(options, printf_callback, printf_stream);
   cs_option(cs_handle, CS_OPT_SYNTAX, ops.intel_syntax ? CS_OPT_SYNTAX_INTEL : CS_OPT_SYNTAX_ATT);
 
+  // Turn on SKIPDATA mode
```

Comment:
The comment is self-evident.

```suggestion
  // Turn on SKIPDATA mode to skip broken instructions, which are ubiquitous in JIT-generated code.
```

Reply:
OK now?